### PR TITLE
Setup actions

### PR DESCRIPTION
**Pull request by Diegores14**
Github action is added to verify the standard and build the pdf.

### CODE REVIEW

File: bin/notebookgen [hunk 1]
```diff
@@ -22,4 +22,4 @@ if (options.args.length < 1) {
   process.exit(1)
 }
 
-notebookgen(options.args[0], options)
+notebookgen(options.args[0], options.opts())
```
Comment (pin3da) [inline bin/notebookgen:25]: Is this related to changes in `commander`?

**Review by pin3da — APPROVED** [review]
Thanks!

File: bin/notebookgen (hunk 1 above)
Comment (Diegores14) [reply bin/notebookgen:25]: yes, to obtain the options this method is used.
